### PR TITLE
Update direct bank transfer doc

### PR DESCRIPTION
**Pull request by VnMa**
__Description__
Create documentation for api v3 version of `[POST] api/v3/user/direct_bank_transfer_payments`

__Screenshot__
image

image

image

### CODE REVIEW

File: react-page/src/oas_spec/sg/paths/v4.json [hunk 1]
```diff
@@ -90,7 +187,7 @@
         "source": "curl --location --request POST \"$BASE_URL/direct_bank_transfer_payments\" \\\r\n  --header \"Content-Type: application/vnd.api+json\" \\\r\n  --header \"Authorization: Bearer $API_TOKEN\" \\\r\n  --form \"amount=100\" \\\r\n  --form \"external_id=ref_20200101_0001\" \\\r\n  --form \"source_bank_account_id=12345\""
       }],
       "tags": [
-        "Direct Bank Transfer Payments (Coming Soon)"
+        "testing"
```
Comment (VnMa) [inline react-page/src/oas_spec/sg/paths/v4.json]: add placeholder value to temporarily hide this api v4 endpoint.

File: react-page/src/oas_spec/sg/paths/v4.json [hunk 2]
```diff
@@ -168,7 +265,7 @@
         "source": "curl --location --request GET \"$BASE_URL/inter_wallet_payments\" \\\r\n  --header \"Content-Type: application/vnd.api+json\" \\\r\n  --header \"Authorization: Bearer $API_TOKEN\""
       }],
       "tags": [
-        "Direct Bank Transfer Payments (Coming Soon)"
+        "testing"
```
Comment (VnMa) [inline react-page/src/oas_spec/sg/paths/v4.json]: add placeholder value to temporarily hide this api v4 endpoint.

File: react-page/src/oas_spec/sg/paths/v4.json [hunk 3]
```diff
@@ -82,6 +82,103 @@
       }
     }
   },
+  "/user/direct_bank_transfer_payments": {
```
Comment (donbader) [inline react-page/src/oas_spec/sg/paths/v4.json]: `api/v3/user/direct_bank_transfer_payments`

Actually this should be inside `v3.json`

File: react-page/src/oas_spec/sg/paths/v4.json [hunk 4]
```diff
@@ -90,7 +90,7 @@
         "source": "curl --location --request POST \"$BASE_URL/direct_bank_transfer_payments\" \\\r\n  --header \"Content-Type: application/vnd.api+json\" \\\r\n  --header \"Authorization: Bearer $API_TOKEN\" \\\r\n  --form \"amount=100\" \\\r\n  --form \"external_id=ref_20200101_0001\" \\\r\n  --form \"source_bank_account_id=12345\""
       }],
       "tags": [
-        "Direct Bank Transfer Payments (Coming Soon)"
+        "cutting_edge"
```
Comment (donbader) [inline react-page/src/oas_spec/sg/paths/v4.json]: ? Why do u change to this...

File: react-page/src/oas_spec/sg/paths/v3.json [hunk 5]
```diff
@@ -3368,6 +3368,103 @@
...
   },
+  "/api/v3/user/direct_bank_transfer_payments": {
+    "post": {
+      "operationId": "createDirectBankTransferPayment",
+      "x-code-samples": [{
+        "lang": "Shell_curl",
+        "source": "curl --location --request POST \"{base_url}\/user\/direct_bank_transfer_payments\" \\\r\n  --header \"X-XFERS-USER-API-KEY: {{user-api-token}}\" \\\r\n--form \"amount=3000.00\" \\\r\n--form \"external_id=foo_bar_123\""
+      }],
+      "tags": [
+        "Direct Bank Transfer Payments (Coming Soon)"
```
Comment (donbader) [inline react-page/src/oas_spec/sg/paths/v3.json]: The tag here is wrong, right?

**Pull request merged**